### DISCUSSION TITLE
2.0.0 release for Forge

### DISCUSSION THREAD

**Discussion by Draylar** [Draylar/identity#301] [General]
I knew I had to give Arch a try at some point, and here we are! 1.17.1 Identity for Forge. Requires the [Architectury API for Forge](https://www.curseforge.com/minecraft/mc-mods/architectury-forge). This release is mega beta and will almost certainly break in some way. Enjoy, and let me know when you encounter bugs!

https://www.curseforge.com/minecraft/mc-mods/identity-on-forge (pending approval)

<hr /><em>This discussion was created from the release <a href='https://github.com/Draylar/identity/releases/tag/2.0.0-1.17.1-forge'>2.0.0 release for Forge</a>.</em>

**Comment by Draylar** [comment]
p.s. this doesn't imply anything for the Fabric version, which will still be updated, and I don't plan to do this for my other Fabric mods at the moment - this was mostly just for fun. Theoretically, both versions will be able to have feature parity at all times, which should make it easy to update on both ends! Fabric will be rolling out with the architectury release once I know the Forge version is stable (& the API will be entirely merged in 1.18).

**Comment by Ch1216** [comment]
Hey just so you know you should mark the Architectury API as a dependency on the curseforge website for the files

**Comment by Zombadee09** [comment]
Can you please make a 1.18 version?
